front/app/containers/Authentication/steps/EmailAndPasswordSignUp/index.tsx

```diff
@@ -99,9 +99,11 @@ const EmailAndPasswordSignUp = ({
       });
       setProfanityApiError(false);
     } catch (e) {
-      const profanityApiError = e.errors.base.find(
-        (apiError) => apiError.error === 'includes_banned_words'
-      );
+      const profanityApiError = Array.isArray(e.errors.base)
+        ? e.errors.base.find(
+            (apiError) => apiError.error === 'includes_banned_words'
+          )
+        : null;
```

Comment:
The rest of the function wouldn't run after we were performing Array#find on undefined. We definitely need tests for this, too. Will add later.